src/pages/Minji/Main/Join.js

```diff
@@ -1,21 +1,153 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Join.scss';
 import { useNavigate } from 'react-router-dom';
-import JoinLogo from '../../image/join로고.png';
+import { upload } from '@testing-library/user-event/dist/upload';
 
 const Join = () => {
+  //email,password정보 저장
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [checkpwd, setCheckPwd] = useState('');
+  const [nickname, setNickName] = useState('');
+  const savenickname = (event) => {
+    setNickName(event.target.value);
+  };
+  const saveemail = (event) => {
+    setEmail(event.target.value);
+  };
+  const savepassword = (event) => {
+    setPassword(event.target.value);
+  };
+
+  const checkpassword = (event) => {
+    setCheckPwd(event.target.value);
+  };
+  //file upload 시 files속성에 저장
+  const [files, setFiles] = useState('');
+
+  const savefile = (e) => {
+    setFiles(e.target.files);
+  };
+  const handleFileChange = (event) => {
+    console.log(event);
+    const selectedFile = event.target.files[0];
+    if (!!selectedFile) {
+      setFiles(selectedFile.name);
+    }
+  };
+  const signiup = () =>
+    fetch('http://10.58.52.85:8000/signUp', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        if (password.length < 10) {
+          alert('비밀번호는 10글자 이상으로 설정해주세요!');
+        } else if (data.message === 'userCREATED') {
+          alert('회원가입 성공');
+          navigate('/minji.donejoin');
+        }
+        console.log(data);
+      });
+
+  //필수데이터 pwd 10글자이상,email'@'','포함 & pwd확인도 일치한지 확인
+  const requiredData =
+    password.length >= 10 &&
+    email.includes('@') &&
+    email.includes('.') &&
+    password === checkpwd;
+
+  const navigate = useNavigate();
+  const goToBack = () => {
+    navigate('/');
+  };
+
   return (
-    <div className="joindone">
-      <div className="back">
-        <span>뒤로</span>
+    <div className="main">
+      <button className="back" type="button" onClick={goToBack}>
+        &lt; &nbsp;&nbsp;뒤로
+      </button>
+      <p>회원가입</p>
+      <div className="basic">
+        <div className="information">기본 정보</div>
+        <div className="required">필수 사항</div>
+      </div>
+      <div className="input_main">
+        <input
+          value={email}
+          onChange={saveemail}
+          type="text"
+          placeholder="이메일"
+        ></input>
```

Comment:
content가 없는태그에 대해서는 모두 self closing적용해주세요
```suggestion
        <input
          value={email}
          onChange={saveemail}
          type="text"
          placeholder="이메일"
         />
```